pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -75,12 +83,39 @@ func (r *rbac) cleanUp(ctx context.Context) error {
 	return nil
 }
 
+func (r *rbac) setDefault() bool {
```

Comment:
@sm43 added defaulting function to set default key and value for param
/cc @nikhil-thomas @vdemeester